internal/deviations/deviations.go

```diff
@@ -95,5 +97,8 @@ var (
 
 	NextHopAFTNotSupported = flag.Bool("deviation_nexthop_aft_not_supported", false, "Device currently doesnot support AFT Next Hop Telemetry. A fully compliant device should support all types of AFT telemetry without this deviation.")
 
+	AllowNilForDefaults = flag.Bool("deviation_allow_nil_for_defaults", false,
```

Comment:
I think "allow" is not really accurate in the name because it's not the device allowing nil, but rather the test is allowing it.

MissingValueForDefaults is probably a better name. We want the deviation to describe the device behavior, not the test behavior.